### DISCUSSION TITLE
Updated Links for TV25 Cincinnati, OH

### DISCUSSION THREAD

**Discussion by blackhern** [iptv-org/community#1115] [Q&A]
Telemundo (WBQCLD)
https://cdn.igocast.com/wkrp_channel1_hls/wkrp_channel1_profile1.m3u8

COZI TV (WBQCLD2)
https://cdn.igocast.com/wkrp_channel2_hls/wkrp_channel2_profile1.m3u8

Outlaw (WBQCLD3)
https://cdn.igocast.com/wkrp_channel3_hls/wkrp_channel3_profile1.m3u8

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-add-a-new-stream-link-to-a-playlists
